### PR TITLE
24 generate email notification

### PR DESCRIPTION
**Pull request by gerauf**

### CODE REVIEW

File: .byebug_history [hunk 1]
```diff
@@ -1,3 +1,23 @@
+c
```
Comment (magsallen) [inline .byebug_history:1]: Let's add .byebug_history to our git ignore

File: db/migrate/20160608183754_create_delayed_jobs.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+class CreateDelayedJobs < ActiveRecord::Migration
+  def self.up
```
Comment (magsallen) [inline db/migrate/20160608183754_create_delayed_jobs.rb:2]: Should we be using def change instead of  def up and down?

Comment (gerauf) [inline db/migrate/20160608183754_create_delayed_jobs.rb:2]: Dont think so - was auto generated by delay job gem

File: spec/features/sending_confirmation_email_spec.rb [hunk 3]
```diff
@@ -0,0 +1,24 @@
+feature "setting up an iou" do
+  context "when creating iou" do
+    scenario "sends an email to the uf" do
+      sign_up
+      expect{create_iou}.to change{ActionMailer::Base.deliveries.size}.by 1
+      iou = Iou.first()
+      invite_email = ActionMailer::Base.deliveries.last
+      expect(invite_email.to).to eq ['youidiot@gmail.com']
```
Comment (magsallen) [inline spec/features/sending_confirmation_email_spec.rb]: change this email address?

File: spec/features/sending_confirmation_email_spec.rb [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+      expect(invite_email.body).to have_content 'Accept PMNB'
+    end
+  end
+
+  context "replying to initial email" do
+    scenario "uf clicks through from email" do
```
Comment (magsallen) [inline spec/features/sending_confirmation_email_spec.rb]: "uf" isn't a very clear payee name

File: spec/helpers/application_helper.rb [hunk 5]
```diff
@@ -9,7 +9,7 @@ def sign_up username: 'Lucy', email: 'lucy@lucy.com',
     click_button 'Sign up'
   end
 
-  def create_iou title: 'Pizza party', amount: 20,  name: "Noah", contact: "07123456789"
+  def create_iou title: 'Pizza party', amount: 20,  name: "Noah", contact: "youidiot@gmail.com"
```
Comment (magsallen) [inline spec/helpers/application_helper.rb]: change email address?

**Comment by magsallen** [comment]
:shipit:

**Pull request merged**